src/app-layout/__tests__/desktop.test.tsx

```diff
@@ -235,8 +233,18 @@ describeEachAppLayout({ sizes: ['desktop'] }, ({ theme }) => {
   test('should render badge when defined', () => {
     const { wrapper } = renderComponent(<AppLayout drawers={manyDrawers} />);
 
-    expect(isDrawerTriggerWithBadge(wrapper, manyDrawers[0].id)).toEqual(true);
-    expect(isDrawerTriggerWithBadge(wrapper, manyDrawers[1].id)).toEqual(false);
+    expect(wrapper.findDrawerTriggerById(manyDrawers[0].id)).toBeTruthy();
+    expect(wrapper.findDrawerTriggerWithBadgeById(manyDrawers[0].id)).toBeTruthy();
+    expect(wrapper.findDrawerTriggerById(manyDrawers[1].id)).toBeTruthy();
+    expect(wrapper.findDrawerTriggerWithBadgeById(manyDrawers[1].id)).toBeFalsy();
+  });
+
+  test('show throw an error when searching for a badge on non-existing drawer', () => {
```

Comment:
Nitpick: double verb ("show throw"), you probably meant just "throw"

Reply:
Nice catch, fixed